projects/step-core/src/lib/modules/table/components/table/table.component.scss

```diff
@@ -1,10 +1,30 @@
-:host {
-  position: relative;
-  display: block;
+step-table {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
 
```

Comment:
According to the tasks description - the issue was about controls which are displayed above the table.
On the first glance, the requested behavior can be achieved just by adding a single line in the `host` rule
```
:host {
   position: relative;
   display: block;
  
   // this line is need to be added
   min-width: fit-content;
}
```

What benefits PR's approach gives in comparison with solution, that is shown above?